app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    first_file, second_file = command[3:].split(" ")
```

Comment:
before that you should check that it won't cause an error
* check that the  `command` length is equal to 3
* check that the first element is `cp`